.checkstyle/checkstyle.xml

```diff
@@ -87,21 +87,23 @@
         </module>
 
         <!-- code quality -->
-        <module name="MethodLength"/>
+        <module name="MethodLength">
+            <property name="max" value="170"/>
+        </module>
         <module name="ParameterNumber">
             <!-- default is 8 -->
         </module>
         <module name="ClassDataAbstractionCoupling">
             <!-- default is 7 -->
-            <property name="max" value="12"/>
+            <property name="max" value="15"/>
         </module>
         <module name="BooleanExpressionComplexity">
             <!-- default is 3 -->
         </module>
 
         <module name="ClassFanOutComplexity">
             <!-- default is 20 -->
-            <property name="max" value="36"/>
+            <property name="max" value="40"/>
```

Comment:
I was always trying to not changing the default metrics value to suppress locally or fixing the problems why now we are doubling these values? :-)